src/compiler/reflection.jl

```diff
@@ -131,6 +131,21 @@ end
 
 const code_ptx = code_native
 
+"""
+    CUDA.return_type(f, tt) -> r::Type
+
+Return a type `r` such that `f(args...)::r` where `args::tt`.
+"""
+function return_type(@nospecialize(func), @nospecialize(tt); kernel::Bool=false,
+                     minthreads=nothing, maxthreads=nothing, blocks_per_sm=nothing,
+                     maxregs=nothing)
```

Comment:
I just copied these parameters from `code_typed`. But are they used for inference?